azure-pipelines.yml

```diff
@@ -1,35 +1,45 @@
 trigger:
+- dev-it
 - master
 - dev-remote-ci
 
 jobs:
-
 - job: 'Ubuntu_16_04'
-  pool:
-    vmImage: 'Ubuntu 16.04'
-  strategy:
-    matrix:
-      Python36:
-        PYTHON_VERSION: '3.6'
+  pool: 'NNI CI GPU'
 
   steps:
-  - script: python3 -m pip install --upgrade pip setuptools
+  - script: python3 -m pip install --upgrade pip setuptools --user
     displayName: 'Install python tools'
   - script: |
       source install.sh
     displayName: 'Install nni toolkit via source code'
+  - script: |
+      python3 -m pip install scikit-learn==0.20.0 --user
```

Comment:
if for test example, we need to install lightgbm and pandas.

Reply:
Currently only test following examples:
mnist
mnist-annotation
mnist-keras
cifar10-pytorch
sklearn-classification
sklearn-regression

When a new example is added,  we should add dependencies here for the new example.